service/README.md

```diff
@@ -0,0 +1,35 @@
+# LitAR Service
+
+LitAR's main codebase is written in Python, the calculation is accelerated by Numba with CUDA supports. To get the best performance, we recommend using edge devices with CUDA cores. We tested our code on a [Nvidia Jetson AGX Xavier developer kit](https://www.nvidia.com/en-us/autonomous-machines/embedded-systems/jetson-agx-xavier/).
+
+## How to Use the Code
+
+Our Python dependencies are managed by [Pipenv](https://pipenv.pypa.io/en/latest/), please follow the [installation instructions](https://pipenv.pypa.io/en/latest/#install-pipenv-today) on its official document to install it first.
+
+Next, activate your environment and install dependencies.
+
+```bash
+pipenv shell
+pipenv install
+```
+
+Start the service.
+
+```bash
+./launch.py serve
+```
+
+## Directory Structure
+
+- `etc`: datasets definitions and loaders.
+- `litar`: LitAR system code.
+- `pyreality`: a 3D computation library.
+- `tests`: testing code.
+- `tmp`: a local temporary file folder (DO NOT REMOVE).
+- `config.py`: configuration file.
+- `launch.py`: command line entry script
+- `Pipfile`: pipenv dependency definition file
+
```

Comment:
Add a dependency graph that shows how each component in the service interacts? The dependency can help others quickly identify which code to modify.

Reply:
Using this pipfile should be enough for understanding the package dependencies. I don't see noticeable benifits of describing the dependencies in a graph format.